builder/metadata.go

```diff
@@ -172,9 +172,19 @@ Latest upstream in format: %d
 // upstream version is 0, then the latest upstream version in the current
 // upstream format will be taken instead.
 func (b *Builder) UpdateVersions(nextMix, nextUpstream uint32) error {
-	format, _, latest, err := b.getUpstreamFormatRange(b.UpstreamVer)
-	if err != nil {
-		return err
+	var format string
+	var latest uint32
+	var err error
+	// Only check online if an upstream version was passed in
```

Comment:
Can you update this comment to reflect what the code is actually doing? Or remove the comment, since the code below seems pretty understandable.

Reply:
sure I think we can remove this